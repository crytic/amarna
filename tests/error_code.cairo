func _safe_transfer{
        syscall_ptr: felt*,
        pedersen_ptr: HashBuiltin*,
        range_check_ptr
    }(
        from_: felt,
        to: felt,
        token_id: Uint256,
        data_len: felt,
        data: felt*
    ):
    _transfer(from_, to, token_id)
    let (success) = _check_onERC721Received(from_, to, token_id, data_len, data)
    with_attr error_message("ERC721: transfer to non ERC721Receiver implementer"):
        assert_not_zero(success)
    end
    return ()
end