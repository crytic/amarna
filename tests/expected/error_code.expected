[sucess-must-be-checked] in error_code.cairo:13:10
[unused-function] in error_code.cairo:1:1